fashify/src/components/Nav/Nav.jsx

```diff
@@ -1,18 +1,21 @@
 import "./Nav.css";
 import logo from "../../logos/hero-logo.png";
-import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
 import { useAuth } from "../../contexts/auth-context";
+import { useData } from "../../contexts/data-context";
```

Comment:
Suggestion: can use barrel export

so that you can directly import from `../contexts`

Reply:
Thaks for the review